### DISCUSSION TITLE
file.lua:1: unexpected symbol near '?'

### DISCUSSION THREAD

**Discussion by lucanori** [cc-tweaked/CC-Tweaked#1334] [Q&A] [answered]
Hello there, first time using git to ask for code, I'm sorry in advance for bad formatting or similar.

I get the error " file.lua:1: unexpected symbol near '?' " when i try to launch a .lua file at the startup of an advanced computer.

the startup.lua is as follow:
`shell.run("file")`

the file.lua is as follow:
```
while(true) do
    http.post("LINK", 'BODY', { ["Content-Type"] = "application/json"})
    sleep(60)
end
```

I want to clarify that if I run the http.post command standalone using the lua interface on computer works just fine.
I need this file to run as soon as the server is online cause I need it to send the status "ONLINE" of the server to home assistant and telegram using http request.

I hope I've given to you everything needed, also there's no "?" in my code, I just checked.

Thank you all!

**Comment by SquidDev** [comment, marked as answer]
Which version of Minecraft are you on? If you're on 1.16.5+, you might be able to update to the latest version of CC:T, which will give you slightly better error messages.

The only thing I can think of which might be happening here is that there's some invisible or broken character on the first line of your file (which is displayed as `?` instead) - you could try deleting the whole line and retyping it out. Sorry, realise that's not very helpful!

**Reply by lucanori** [reply]
Actually, it was in fact true. There was a "?” at the start of the line, that VS Code wasn't showing
